File: evidence/generated-systems/angular/aerospace-on-angular/src/app/models/AircraftOption.js

// Define collection and schema for AircraftOption
export interface AircraftOption {
    code:
	type : string
    name:
	type : string
    Variants:
 	type : [{ type: Schema.Types.ObjectId, ref: 'AircraftVariant' }]
    Packages:
 	type : [{ type: Schema.Types.ObjectId, ref: 'AircraftPackage' }]
    OptionCategory:
 	type : String
#
    collection: 'aircraftOptions'
}

File: evidence/generated-systems/rubyonrails/aerospace-on-rails/aerospaceonrails/app/models/aircraftoption.rb
class AircraftOption < ApplicationRecord
  enum OptionCategory: [:Cabin, :Connectivity, :Safety, :Performance, :Paint, :FlightDeck]


  has_many :Variants, class_name: 'AircraftVariant'
  has_many :Packages, class_name: 'AircraftPackage'

end
